coalib/collecting/Collectors.py

```diff
@@ -284,13 +285,9 @@ def collect_registered_bears_dirs(entrypoint):
     :param entrypoint: The entrypoint to find packages with.
     :return:           List of bear directories.
     """
-    collected_dirs = []
-    for ep in pkg_resources.iter_entry_points(entrypoint):
-        registered_package = None
-        try:
-            registered_package = ep.load()
-        except pkg_resources.DistributionNotFound:
-            continue
-        collected_dirs.append(os.path.abspath(
-            os.path.dirname(registered_package.__file__)))
-    return collected_dirs
+    eps = {find_loader("bears")
```

Comment:
Why loop over entrypoints and create `ep` is `ep` is not being used in the `find_loader("bears")` part ??
So, `eps` will just be a set of the same value repeated ?